src/client.ts

```diff
@@ -1258,15 +1261,68 @@ export class MatrixClient extends EventEmitter {
         this.supportsCallTransfer = support;
     }
 
+    public async getLocalVideoStream() {
```

Comment:
This is an artifact of the [original group call implementation](https://github.com/vector-im/matrix-video-chat/blob/4d7e5583eb055d0fa2940ac0201f344e70cab588/src/Room.jsx#L232) in matrix-video-chat. I used this method to preview the user's local video stream. I think with the addition of `groupCall.initLocalParticipant()` these public methods are no longer necessary. However, we should still keep track of the `localAVStream`. We want to reuse the `localAVStream` and call `localAVStream.clone()` to get the streams to use in additional calls. In Safari, without calling `localAVStream.clone()` you'll get a black video.

Reply:
I think it would make sense to move this to the [`MediaHandler`](https://github.com/SimonBrandner/matrix-js-sdk/commit/a1f73eee86aff8b5b3c667573b44942c3c4e23f9) which will be introduced by https://github.com/matrix-org/matrix-js-sdk/pull/1911